.github/workflows/ci.yml

```diff
@@ -1,9 +1,12 @@
 name: Run CI
 on:
   push:
-    branches: ["**"]
+    branches: ["master"]
```

Comment:
do we need the push trigger if it is used only for master which is protected by checks?

Reply:
I guess this provides an extra layer of protection against build breakage and allows showing the checks passing in the commit message in GitHub.